swagger.yaml

```diff
@@ -139,16 +132,20 @@ parameters:
     required: true
     description: '"Bearer [token]" where [token] is your OAuth2 access token'
 definitions:
-  ResultObject:
+  ResultObjects:
     properties:
       data:
         type: array
         items:
           $ref: "#/definitions/ResourceObject"
+  ResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/ResourceObject"
   ResourceObject:
     properties:
       id:
-        type: integer
+        type: string
       type:
         type: string
         example: "directory"
```

Comment:
Instead of using `example`, can you use `enum: [directory]` instead?

Reply:
Done